.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Pull Request Template
```

Comment:
Personally, I am not willing to add this template, because it may dissuade some developers; unlike issues, anyone, as long as it is willing to submit a pr, I will review it.

Reply:
understandable and acceptable. I'll remove the template.